make theme_color of Manifest can be set

bekcpear opened the pull request:
The 'theme_color' of Manifest affects the color of status bar
of mobile view, however it's static before. Make it can be set
via `filebrowser config set --branding.color "<hex-color-code>"`.

Signed-off-by: bekcpear <i@bitbili.net>

bekcpear committed:
make theme_color of Manifest can be set

The 'theme_color' of Manifest affects the color of status bar
of mobile view, however it's statc before. Make it can be set
via `filebrowser config set --branding.color "<hex-color-code>"`.

Signed-off-by: bekcpear <i@bitbili.net>

ramiresviana reviewed (changes requested):
Chrome desktop/mobile ignores the theme-color from dynamic manifest. You have to use template variables on [index.html#L19](https://github.com/filebrowser/filebrowser/blob/3768e3345f071c9a7ae80b7c75092000df9afc1b/frontend/public/index.html#L19) and [index.html#L29](https://github.com/filebrowser/filebrowser/blob/3768e3345f071c9a7ae80b7c75092000df9afc1b/frontend/public/index.html#L29).

bekcpear committed:
set the theme-color and msapplication-TileColor in the index.html either

Thanks @ramiresviana for pointing out the problem:
"Chrome desktop/mobile ignores the theme-color from dynamic manifest."
